docs/wait.md

```diff
@@ -32,6 +32,9 @@ time([<span class="tag fun"></span>](types.md#function)&nbsp;toRunFunc, [<span c
     * [<span class="tag fun"></span>](types.md#function) **timeoutFunc**: The function that that triggers if the timeout amount is met.
         * {>>Optional, defaults to no function being triggered if a timeout happens.<<}
 
+!!!warning
+    **conditionFunc** is checked every frame, try to make this function as simple as possible to avoid performance issues. If complicated logic here is required, [Wait.time()](#time) can be used as a more controllable alternative.
```

Comment:
> If complicated logic here is required

Got a little bit of Yoda action going there 😉 If you don't mind, could this please be updated to:

> If complicated logic is required here

Reply:
While you're at it, if you want to change "controllable" to "flexible" that'd be great. Just being picky though, I wouldn't let something like that on its own hold up a PR 😉